invisible_cities/evm/event_model.py

```diff
@@ -192,14 +192,18 @@ class Voxel(BHit):
     """Represents a Voxel"""
     def __init__(self, x,y,z, E, size, hits=[], e_type : HitEnergy = HitEnergy.E.value):
         super().__init__(x,y,z, E)
-        self._size = size
-        self.hits = hits
+        self._size  = size
+        self.hits   = hits
+        self.e_type = e_type
 
     @property
     def size(self): return self._size
 
     @property
-    def Ehits(self): return sum(getattr(h, e_type) for h in hits)
+    def Ehits(self): return sum(getattr(h, self.e_type) for h in hits)
+
+    @property
```

Comment:
What is the purpose of this property? There already is a public attribute containing exactly the same information. Why the duplication?

Reply:
If your question is what is the difference between `Ehits`and `energy`, the answer is that the voxel energy is set in the voxelization algorithm, coming from the numpy histogram function, and, although it should be the same as the sum of the hit energies, I wanted to keep it separated as a mean of checking that the numpy histogram function keeps doing its job.

Reply:
Sorry, I don't understand how to use GitHub's code comment feature: I click on the line I want to comment on, and it highlights some other lines nearby!

I was talking about this:

    def Etype(self): return self.e_type

With this, `x.Etype` and `x.e_type` are 100% synonymous (though the latter is settable, while the former is not). What is the point of this duplication?

(The same happens in `Cluster` too.)

Reply:
Ok, this is an issue we've been talking about for some time, and we need to decide a unique way of using properties and attribute in the IC event model (right now it's confusing). However, I think that this issue deserves another PR.